libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -855,7 +855,9 @@ export function shouldIncludeSelfForRepetitionReference(manifest: OperationManif
 }
 
 export function loadParameterValue(parameter: InputParameter): ValueSegment[] {
-  const valueObject = parameter.isNotificationUrl ? `@${constants.HTTP_WEBHOOK_LIST_CALLBACK_URL_NAME}` : parameter.value;
+  const valueObject = parameter.isNotificationUrl
```

Comment:
At this point can we break this down to an actual if/else, mixing ternaries with pretty much anything makes readability difficult. (I say this even as a ternary expression apologist)

Reply:
Yup, will do. I tried to break it down with `()`s but Prettier was having none of that.